sdk/eventhub/Microsoft.Azure.Management.EventHub/src/Microsoft.Azure.Management.EventHub.csproj

```diff
@@ -9,12 +9,12 @@
 			Provides developers with a library to create and manage all Azure Event Hubs resources. Note: This client library is for EventHub under Azure Resource Manager.
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.EventHubs" (https://www.nuget.org/packages/Azure.ResourceManager.EventHubs/). Please see the package changelog for more information.
 		</Description>
-		<Version>4.0.0</Version>
+		<Version>5.0.0</Version>
```

Comment:
Why do we use a stable version number here when the SDK is generated from a preview swagger spec?

Reply:
Aren't api version bump ups done when there are breaking changes or introduction of new features ?

Reply:
Yes, we should. I have this question more because I'm curious why you don't use a preview version like `4.1.0-preview`. But anyway, I'm ok with the current change.

Reply:
Okay, that actually makes sense, I will consult our manager and see if this works for us.